diagnostic_feedback/public/js/src/student/student_quiz.js

```diff
@@ -144,15 +145,42 @@ function StudentQuiz(runtime, element, initData) {
 
           if (success && response.student_result) {
             //log event for quiz finish
-            event_data.event_type = 'xblock.diagnostic_feedback.quiz.finish';
+
+            var event_data = {
```

Comment:
@jareerahsan If you declare `event_data` outside of this block (`var event_data;`), you can just assign to it here and below (`event_data = ...`). No need to declare it multiple times.